src/vs/workbench/api/browser/mainThreadEditorTabs.ts

```diff
@@ -69,7 +71,8 @@ export class MainThreadEditorTabs {
 				tabs.push({
 					group: group.id,
 					name: editor.getTitle(Verbosity.SHORT) ?? '',
-					resource: editor.resource
+					resource: editor.resource,
+					isActive: this._editorService.activeEditor?.resource?.toString() === editor.resource.toString()
```

Comment:
I believe you can also figure this out via `IEditorGroupsService#activeGroup` and `IEditorGroup#isActive(...)`

Reply:
Thanks, I didn't notice that.